cybershell/kb.py

```
@@ -0,0 +1,23 @@
+from dataclasses import dataclass
+from typing import List
+from .memory import NeuralMemory, MemoryItem
+
+@dataclass
+class KBEntry:
+    title: str
+    content: str
+    tags: List[str]
+
+class KnowledgeBase:
+    def __init__(self):
```

Ruff (0.12.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)